design/object-bucket/object-bucket.md

```diff
@@ -183,56 +165,7 @@ status:
 1. `configMapRef` is an objectReference to the generated ConfigMap 
 1. `secretRef` is an objectReference to the generated Secret
 
-### OB Custom Resource Definition
-
-```yaml
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: objectbuckets.store-operator.s3
-spec:
-  group: store-operator.s3
-  names:
-    kind: ObjectBucket
-    listKind: ObjectBucketList
-    plural: objectbuckets
-    singular: objectbucket
-  scope: Namespaced
-  version: v1alpha1
-  subresources:
-    status: {}
-```
-
-### OB Custom Resource
-
-```yaml
-apiVersion: store-operator.s3/v1alpha1
-kind: ObjectBucket
-metadata:
-  name: object-bucket-claim-MY-BUCKET-1
-  ownerReferences: [1]
-  - name: CEPH-CLUSTER
-    ...
-  labels:
-    ceph.rook.io/object: [2]
-spec:
-  objectBucketSource: [4]
-    provider: ceph.rook.io/object
-status:
-  claimRef: objectreference [5]
-  phase: {"pending", "bound", "lost"} [6]
-```
-
-1. `ownerReferences` makes the OB a child of the object store.  If the store is destroyed, the OBs are deleted.
-1. Label `ceph.rook.io/object/claims` associates all artifacts under the Rook-Ceph object provisioner.
-1. `objectBucketSource` is a struct containing metadata of the object store provider
-1. `claimRef` is an objectReference to the ObjectBucketClaim associated with this ObjectBucket
-1. `phase` is the current state of the ObjectBucket
-    - `pending`: the operator is processing the request
-    - `bound`: the operator finished processing the request and linked the OBC and OB
-    - `lost`: the OBC has been deleted, leaving the OB unclaimed
-
-### (User Access Key) Secret
+### Secret (bucket lib generated, consumed by pod)
 
 ```yaml
 apiVersion: store-operator.s3/v1alpha1
```

Comment:
API group name has since changed to `objectbucket.io`.  version remains the same